public/client/TracyCallstack.cpp

```diff
@@ -401,16 +468,29 @@ CallstackSymbolData DecodeSymbolAddress( uint64_t ptr )
 
 CallstackEntryData DecodeCallstackPtr( uint64_t ptr )
 {
+    InitRpmalloc();
+
+    const ModuleNameAndBaseAddress moduleNameAndAddress = GetModuleNameAndPrepareSymbols( ptr );
```

Comment:
This is outside dbghelp lock.

Reply:
I've moved the locking up to include it.
Maybe it would be better to have a scoped wrapper for this lock though.